conduct/code_of_conduct.md

```diff
@@ -0,0 +1,134 @@
+   communicating with someone with a different primary language or cultural
+   background.
+
+4. **Be respectful**. Not all of us will agree all the time, but disagreement is
+   no excuse for poor behavior or poor manners. We might all experience some
+   frustration now and then, but we cannot allow that frustration to turn into a
+   personal attack. It’s important to remember that a community where people
+   feel uncomfortable or threatened is not a productive one.
+
+5. **Be careful in the words that you choose**. Be kind to others. Do not insult
+   or put down other community members. Harassment and other exclusionary
+   behavior are not acceptable. This includes, but is not limited to:
+   * Violent threats or violent language directed against another person
+   * Discriminatory jokes and language
+   * Posting sexually explicit or violent material
+   * Posting (or threatening to post) other people's personally identifying
+     information ("doxing")
+   * Personal insults, especially those using racist or sexist terms
+   * Unwelcome sexual attention
+   * Advocating for, or encouraging, any of the above behavior
+   * Repeated harassment of others. In general, if someone asks you to stop,
+     then stop
+
+6. **Moderate your expectations**. Many in our community volunteer their time
+   and/or have multiple responsibilities. It's unlikely they are purposefully
+   ignoring issues, refusing to engage in discussion, avoiding features, etc.
+   Chances are, they need to attend to other aspects of their lives.
+
+7. **When we disagree, try to understand why**. Disagreements, both social and
+   technical, happen all the time and Jupyter is no exception.  Try to
+   understand where others are coming from, as seeing a question from their
+   viewpoint may help find a new path forward.  And don’t forget that it is
+   human to err: blaming each other doesn’t get us anywhere, while we can learn
+   from mistakes to find better solutions.
+
+8. **A simple apology can go a long way**. It can often de-escalate a situation,
+   and telling someone that you are sorry is act of empathy that doesn’t
+   automatically imply an admission of guilt.
+
+
+## Reporting
+
+If you believe someone is violating the code of conduct, please report this in
+a timely manner. Code of conduct violations reduce the value of the community
+for everyone and we take them seriously.
+
+You can file a report by emailing
+[*conduct@jupyter.org*](mailto:conduct@jupyter.org) or by filing out
+[this form](https://goo.gl/forms/sJzOIie3zde9M71T2). For more details or
+information on reporting in-person at an event, please see our Reporting
+Guidelines.
+
+The online form gives you the option to keep your report anonymous or request
+that we follow up with you directly. While we cannot follow up on an anonymous
+report, we will take appropriate action.
+
+
+## Enforcement
+
+For information on enforcement, please view the [*Enforcement
+Manual*](enforcement.md).
```

Comment:
Actually the opposite :)  I meant to rename all files to remove the otherwise redundant `conduct_` prefix, since they're already in a folder named that. Thx for the reminder!